ccd-definition/CaseField/CareSupervision/manageDocumentsLA.json

```diff
@@ -43,5 +43,24 @@
     "Label": "Which hearing?",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "furtherEvidenceTypeListLA",
```

Comment:
maybe better name required, see design in https://tools.hmcts.net/jira/browse/FPLA-2763, new label on first radio button is 'Further documents for main application', maybe this could be 'furtherDocumentsTypeListLA' - then furtherEvidenceTypeListLA can be used for the further evidence subtype question within collection (2nd story after this one)